src/components/common/EmulatorDisabled.tsx

```diff
@@ -59,6 +59,43 @@ export const EmulatorDisabled: React.FC<
   </>
 );
 
+export const ExtensionsEmulatorDisabled: React.FC<
+  React.PropsWithChildren<{}>
```

Comment:
`React.PropsWithChildren<{}>` this should be safe to drop?

Reply:
Removed